doc/specifications/proposed/content_download/content_download.md

```diff
@@ -41,10 +41,12 @@ Example: `/content/download/68/file/My-file.pdf`
 
 - language (optional)
 
+  > Should we keep this, given that the fileId binds to a particular language ?
```

Comment:
guess since implementation does not use it anymore it should be removed from spec until we see a need for it.